### PR TITLE
chore: add labels and error to the gcp_resources.proto

### PR DESCRIPTION
**Pull request by helinwang**
- added the error field, which is present in the backend proto definition
- added the labels field, user can use it to save custom metadata

**Description of your changes:**

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: components/google-cloud/google_cloud_pipeline_components/experimental/proto/gcp_resources.proto [hunk 1]
```diff
@@ -12,8 +14,12 @@ message GcpResources {
     // The unique resource uri. E.g.
     // https://dataflow.googleapis.com/v1b3/projects/project_1/locations/us-central1/jobs/123
     optional string resource_uri = 2;
+    // The error from the resource.
+    google.rpc.Status error = 3;
+    // The labels of the resource, can be used to save custom metadata.
```
Comment (IronPan) [inline components/google-cloud/google_cloud_pipeline_components/experimental/proto/gcp_resources.proto:19]: please update the comment.

Comment (helinwang) [reply components/google-cloud/google_cloud_pipeline_components/experimental/proto/gcp_resources.proto]: thanks, done.

**Comment by helinwang** [comment]
If I recall correctly, Rui took over this and sent another CL. Closing this now.

**Pull request closed without merging**